From 593faa093bdd60cd562d347fdfee26e20570ff30 Mon Sep 17 00:00:00 2001
From: Martin Pitt <martinpitt@gnome.org>
Date: Fri, 24 May 2013 06:52:03 +0200
Subject: [PATCH 1/2] Clean up remaining byteswap code.

Replace the remaining three CT_LE_* macro calls with htole*, completing commit
f3bd63.

This fixes the build failure:

   boot.o: In function `write_volume_label':
   src/boot.c:560: undefined reference to `cpu_to_le16'
---
 src/boot.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/boot.c b/src/boot.c
index b7c4e4d..86cc682 100644
--- a/src/boot.c
+++ b/src/boot.c
@@ -557,9 +557,9 @@ static void write_volume_label(DOS_FS * fs, char *label)
       de.ctime = de.time;
       de.cdate = de.date;
       de.adate = de.date;
-      de.starthi = CT_LE_W(0);
-      de.start = CT_LE_W(0);
-      de.size = CT_LE_L(0);
+      de.starthi = htole16(0);
+      de.start = htole16(0);
+      de.size = htole32(0);
     }
 
     fs_write(offset, sizeof(DIR_ENT), &de);
-- 
1.8.1.2

